angular-template.js

```diff
@@ -7,5 +7,28 @@ module.exports = {
             Overrides to Angular template recommended rules:
             https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/src/configs/recommended.json
         */
+
+        /*
+            Overrides to Angular rules outside of the recommended configuration:
+        */
+
+        '@angular-eslint/template/accessibility-alt-text': 'error',
+        '@angular-eslint/template/accessibility-elements-content': 'error',
+        '@angular-eslint/template/accessibility-label-for': 'error',
+        '@angular-eslint/template/accessibility-label-has-associated-control': 'error',
+        '@angular-eslint/template/accessibility-table-scope': 'error',
+        '@angular-eslint/template/accessibility-valid-aria': 'error',
+        '@angular-eslint/template/click-events-have-key-events': 'error',
+        '@angular-eslint/template/conditional-complexity': 'error',
```

Comment:
This is worth considering. I follow this rule as a personal practice, because I don't like code in templates. I have definitely seen more complex conditionals in reviews, but I'm not sure anything more than the default for this rule, 4.

Reply:
I agree it seems like a useful rule. Improve readability and testability by moving complex code out of the template and into the component TS.

Reply:
**Proposal**
```javascript
'@angular-eslint/template/conditional-complexity': 'error'
```

Reply:
@TrevorKarjanis this is ready to close